Sources/Shared/Data Manager/Model/Documents/File.h

```diff
@@ -59,5 +59,7 @@
 //Tool method to retrive the type of the File
 + (NSString *)fileType:(NSString *)fileName;
 
+-(NSString *) getNatureName;
+-(NSString *) convertNameToNature;
```

Comment:
Please rename to `getNaturalName` or 'getPrettyName`, `convertToNaturalName` ...